tests/base/data_out_base_vtu.cc

```diff
@@ -22,11 +22,38 @@
 
 #include "patches.h"
 
-// Output data on repetitions of the unit hypercube
 
-// define this as 1 to get output into a separate file for each testcase
-#define SEPARATE_FILES 0
+void filter_out_xml_key(std::stringstream &in, const std::string &key, std::stringstream &out)
```

Comment:
I think you should make the last argument just an `std::ostream` since you're writing into it. Then also make the first argument a `std::istream`.